src/Arduino/Arduino.ino

```diff
@@ -61,7 +60,16 @@ unsigned long currentMillis, prevSendMillis = 0;
 
 // Declare NeoPixel strip object for bar graphs:
 Adafruit_NeoPixel pixels(95, neopixel_pin, NEO_GRB + NEO_KHZ800);
-Button startButton;
+
+//Array of buttons
+int NUMBER_OF_BUTTONS = 5;
```

Comment:
It looks like this value isn't required to instantiate your `buttons` array, so you may be able remove it, and always use the Arduino equivalent of `buttons.length` in your for loop. That way this var won't need to be manually updated each time another button is added to `buttons`.

Reply:
you would think you could do that but it's more complicated than that.
the function sizeof() returns the length of an array, however when it's an array of objects it returns the size of the array in bytes.
You can then divide the returned value by the size in bytes of one object... 
Or you can just update it manually.  :/